cmd/ltctl/loadtest.go

```diff
@@ -5,14 +5,61 @@ package main
 
 import (
 	"fmt"
+	"math"
 	"time"
 
 	"github.com/mattermost/mattermost-load-test-ng/coordinator"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance/prometheus"
 	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform"
 
 	"github.com/spf13/cobra"
 )
 
+func getErrorsInfo(t *terraform.Terraform, status coordinator.Status) (map[string]int64, error) {
+	timeRange := int(time.Since(status.StartTime).Round(time.Second).Seconds())
+	queries := []struct {
+		description string
+		query       string
+	}{
+		{
+			"Timeouts",
+			fmt.Sprintf("sum(increase(loadtest_http_timeouts_total[%ds]))", timeRange),
+		},
+		{
+			"HTTP 5xx",
+			fmt.Sprintf("sum(increase(loadtest_http_errors_total{status_code=~\"5..\"}[%ds]))", timeRange),
+		},
+		{
+			"HTTP 4xx",
+			fmt.Sprintf("sum(increase(loadtest_http_errors_total{status_code=~\"4..\"}[%ds]))", timeRange),
+		},
+	}
+	info := make(map[string]int64, len(queries)+1)
+
+	output, err := t.Output()
+	if err != nil {
+		return info, err
+	}
+
+	prometheusURL := fmt.Sprintf("http://%s:9090", output.MetricsServer.PublicIP)
+	helper, err := prometheus.NewHelper(prometheusURL)
+	if err != nil {
+		return info, fmt.Errorf("failed to create prometheus.Helper: %w", err)
+	}
```

Comment:
Maybe it's better to do this outside in the main function and just pass the helper object instead of `t`.

Reply:
I was trying to keep `RunLoadTestStatusCmdF` as clean as possible but that works too.